spa/src/utils/axiosInstance.ts

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
```

Comment:
I assume only places use this `axiosInstance` is the code within `src/api`? So maybe we should move this file to under the folder `src/api/`?

Reply:
Updated